tests/load/filesystem/test_filesystem_client.py

```diff
@@ -0,0 +1,106 @@
+from typing import Sequence, Tuple, cast, List
+import shutil
+from pathlib import Path
+import json
+
+import pytest
+
+from dlt.common.utils import uniq_id
+from dlt.common.storages import LoadStorage, FileStorage
+from dlt.common.schema import Schema
+from dlt.common.destination.reference import DestinationReference, LoadJob
+from dlt.destinations.filesystem.configuration import FilesystemClientConfiguration
+from dlt.destinations.filesystem.filesystem import FilesystemClient, LoadFilesystemJob
+from dlt.destinations import filesystem
+from dlt.load import Load
+
+from tests.utils import clean_test_storage, init_test_logging, TEST_DICT_CONFIG_PROVIDER, preserve_environ
+from tests.load.filesystem.utils import get_client, setup_loader
+
+
+@pytest.fixture(autouse=True)
+def storage() -> FileStorage:
+    return clean_test_storage(init_normalize=True, init_loader=True)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def logger_autouse() -> None:
+    init_test_logging()
+
+
+NORMALIZED_FILES = [
+    "event_user.839c6e6b514e427687586ccc65bf133f.0.jsonl",
+    "event_loop_interrupted.839c6e6b514e427687586ccc65bf133f.0.jsonl"
+]
+
+
+def test_succesful_load(all_buckets_env: str) -> None:
+    dataset_name = 'test_' + uniq_id()
+    dataset_name = 'test_' + uniq_id()
+    client, jobs, root_path, load_id = perform_load(all_buckets_env, dataset_name, NORMALIZED_FILES, write_disposition='replace')
+
+    for job in jobs:
+        assert job.state() == 'completed'
+        dest_fn = LoadFilesystemJob.make_destination_filename(job.file_name(), client.schema.name, load_id)
+        assert client.fs_client.exists(root_path.joinpath(dest_fn))
+
+
+def test_replace_write_disposition(all_buckets_env: str) -> None:
+    dataset_name = 'test_' + uniq_id()
+    client, jobs1, root_path, load_id1 = perform_load(all_buckets_env, dataset_name, NORMALIZED_FILES, write_disposition='replace')
```

Comment:
something wrong with "test_replace_write_disposition"
Always creates LoadEmptyJob. files. does not raise AccessDenied, is it working on a local file? `perform_load` is always passing

Reply:
I'm not seeing this now. I'm getting fail with wrong aws and gc credentials. But I'll take a better look tomorrow if it's correct.